build/ubuntu-1704/test.sh

```diff
@@ -28,6 +29,26 @@ ls -l /usr/share/ca-certificates.backup/* | wc -l | grep $total
 # Restore
 /bin/cert-manage -restore
 /bin/cert-manage -list | wc -l | grep $total
+
+## Chrome
+echo "Chrome tests"
+timeout 10s chromium-browser --no-sandbox --headless https://google.com 2>&1 >> /var/log/chrome.log
+# /bin/cert-manage -list -app chrome | wc -l
+
```

Comment:
I've disabled chrome as it's using the newer `cert9.db` version. I haven't dug into that enough to know if we can safely use it or not. 

https://wiki.mozilla.org/NSS_Shared_DB#Where_we_are_today